lisa/mixin_modules.py

```diff
@@ -21,9 +21,12 @@
 import lisa.sut_orchestrator.ready  # noqa: F401
 
 if platform.system() == "Linux":
-    import lisa.sut_orchestrator.qemu.context  # noqa: F401
     import lisa.sut_orchestrator.qemu.platform  # noqa: F401
     import lisa.sut_orchestrator.qemu.schema  # noqa: F401
+    import lisa.sut_orchestrator.libvirt.context  # noqa: F401
+    import lisa.sut_orchestrator.libvirt.platform  # noqa: F401
+    import lisa.sut_orchestrator.libvirt.schema  # noqa: F401
+    import lisa.sut_orchestrator.cloud_hypervisor.platform  # noqa: F401
```

Comment:
Can we have a libvirt folder to put all of them in it? Because the libvirt is not an independent platform.

Reply:
You mean it should not be inside `sut_orchestrator` folder? So, something like `lisa/libvirt/context.py"?

Reply:
I mean create a libvirt folder only, and put libvirt, cloud-hypervisor and qemu all in it.

Reply:
Okay, so there will be `lisa/sut_orchestrator/libvirt` and all files will be directly in it?

Reply:
yes